src/test/java/com/exasol/bucketfs/BucketAccessExceptionTest.java

```diff
@@ -0,0 +1,41 @@
+package com.exasol.bucketfs;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.startsWith;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import java.net.URI;
+
+import org.junit.jupiter.api.Test;
+
+public class BucketAccessExceptionTest {
+    private static final URI EXPECTED_URI = URI.create("http://localhost:2580/default");
+    private static final String EXPECTED_MESSAGE = "the message";
+    private static final Throwable EXPECTED_CAUSE = new IllegalArgumentException();
+
+    @Test
+    public void testCreateFromOtherException() throws Exception {
```

Comment:
Is it a Junit 4 test? Junit 5 methods are not public.

Reply:
Fixed.